### PR TITLE
chore: remove prisma-sqlite junk

### PR DESCRIPTION
**Pull request by juliusmarminge**
Ref: #2835 (Closes half of it)

## 🎯 Changes

What changes are made in this PR? Is it a feature or a bug fix?

- Removes all the sqlite junk in favor of a Postgres container - takes ~25sec to boot up. Made some slight changes that allowed the build and test-dev scripts to be parallelized thus saving in some of the added time for the container startup. Total diff not too bad...

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

**Comment by juliusmarminge** [comment]
Can you remove the interop ws example from branch protection @KATT? It doesn't exist anymore

CleanShot 2023-02-18 at 16 38 52

File: .github/workflows/examples-old-nodes.yml [hunk 1]
```diff
@@ -52,27 +65,14 @@ jobs:
       - name: Build trpc
         run: pnpm turbo --filter "@trpc/*" build
 
-      - name: Next.js cache
```
Comment (juliusmarminge) [inline .github/workflows/examples-old-nodes.yml:55]: Turbo does the caching for us so this is not used

File: .github/workflows/examples.yml [hunk 2]
```diff
@@ -26,7 +34,6 @@ jobs:
             express-server,
             express-minimal,
             .interop/next-prisma-starter,
-            .interop/next-prisma-websockets-starter,
```
Comment (juliusmarminge) [inline .github/workflows/examples.yml:29]: Gotta remove this from branch protection

File: .github/workflows/examples-old-nodes.yml [hunk 3]
```diff
@@ -5,6 +5,14 @@ on:
 
 jobs:
   e2e-old-nodes:
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
```
Comment (KATT) [inline .github/workflows/examples-old-nodes.yml:13]: nice, i didn't know this one

**Review by KATT — DISMISSED** [review]
❤️

**Review by KATT — DISMISSED** [review]

File: .github/workflows/examples-old-nodes.yml (hunk 3 above)
Comment (juliusmarminge) [reply .github/workflows/examples-old-nodes.yml:13]: It's the same we use in the docker-compose that comes with a few examples ;)

CleanShot 2023-02-24 at 19 44 12@2x

**Review by KATT — APPROVED** [review]

**Pull request merged**